packages/aws-cdk-lib/aws-stepfunctions/lib/states/state.ts

```diff
@@ -491,13 +491,26 @@ interface ChoiceTransition {
   condition: Condition;
 }
 
+/**
+ * Options for Choice Transition
+ */
+export interface IChoiceTransitionOptions {
```

Comment:
@msambol what was the reason for naming this `IChoiceTransitionOptions`? I assume you ran into some linter rule that yelled at you? Want to find out because I don't think it makes sense here, and we should exempt this from that rule if thats the case

Reply:
@kaizencc Yep, I got:

```
aws-cdk-lib:     The declaring class is introduced here
aws-cdk-lib: aws-stepfunctions/lib/states/state.ts:503:3 - error JSII3008: The "comment" property of struct "aws-cdk-lib.aws_stepfunctions.ChoiceTransitionOptions" must be "readonly". Rename "aws-cdk-lib.aws_stepfunctions.ChoiceTransitionOptions" to "IChoiceTransitionOptions" if it is meant to be a behavioral interface.
```

Reply:
@kaizencc From what I'm reading there isn't a way to silence JSII errors? Hmm..